source/systems/gaea_user_guide.rst

```diff
@@ -91,2113 +93,1545 @@ points at Atlanta and Chicago.
-+---------------+---------------------------------------------------+
-| **CMRS**      | Climate Modeling and Research System; an          |
-|               | alternate name for Gaea.                          |
-+---------------+---------------------------------------------------+
-
-
-Environment
-------------
-
-Gaea is implemented to use the Environment Modules system. This tool
-helps users manage their Unix or Linux shell environment. It allows
-groups of related environment-variable settings to be made or removed
-dynamically. Modules provides commands to dynamically load, remove and
-view software.
-
-More information on using modules is available at Gaea Modules.
-
-Do's and Don'ts
----------------
-
-**Do**
-
-- Compile on login nodes
-- Put transient data in /gpfs/f5/<project>/scratch/$USER
-- Copy data back to archive location (off gaea) using RDTN's
-- Use gcp for transfers
-
-**Don't** use the following on Gaea:
-
-- combines on compute nodes
-- combines on batch (they will be killed)
-- compile on batch
-- cp
-- cron jobs (not permitted)
-- deep large scale use of "find" on the F5 filesystem (please use 'lfs
-  find' instead)
-- fs as permanent storage
-- module purge
-- recursive operations like ls -R
-- run applications natively
-- transfers on batch nodes
-- unalias*
-
-File Systems
-============
+.. _gaea-compiling:
 
-Gaea has three filesystems: Home, F2 (a parallel file system based on
-Lustre, **decommissioned**), and F5 (a General Parallel File System).
-
-Summary of Storage Areas
-------------------------
-
-**NFS File System**
+*********
+Compiling
+*********
 
-.. list-table::
-   :header-rows: 1
-   :stub-columns: 1
-   :align: left
-
-   * - Area
-     - Path
-     - Type
-     - Permissions
-     - Quota
-     - Backedup
-   * - User Home
-     - ``/ncrc/home[12]/$USER``
-     - NFS
-     - User Set
-     - 50 GB
-     - Yes
-   * - Project Home
-     - ``/ncrc/proj/<project>``
-     - NFS
-     - Project Set
-     - N/A
-     - Yes
-
-
-**GPFS (F5)**
+.. _gaea-compilers:
 
-.. list-table::
-   :header-rows: 1
-   :stub-columns: 1
-   :align: left
-
-   * - Area
-     - Path
-     - Type
-     - Permissions
-     - Quota
-     - Backedup
-     - Purged
-   * - Member Work
-     - ``/gpfs/f5/<project>/$USER``
-     - GPFS
-     - User Set
-     - Project-Based
-     - No
-     - No
-   * - Project Work
-     - ``/gpfs/f5/<project>/proj-shared``
-     - GPFS
-     - 750
-     - Project-Based
-     - No
-     - No
-   * - World Work
-     - ``/gpfs/f5/<project>/world-shared``
-     - GPFS
-     - 755
-     - Project-Based
-     - No
-     - No
-
-
-HOME
-----
+Compilers
+=========
 
-The home filesystem is split into two sections both of which are
-backed up. For load balance purposes, there is a home1 and home2.
-Note:
+Cray, AMD, NVIDIA, and GCC compilers are provided through modules on Gaea.
+There is also a system/OS versions of GCC available in :file:`/usr/bin`. The
```

Comment:
did you mean "a system version" or "system versions"?

Reply:
Fixed in new push.